secret_number.js

```diff
@@ -1,4 +1,18 @@
 'use strict';
 module.exports = function() {
-  
+
+let num;
+
+// = Math.floor(Math.random()*1000001);
+
+(function randomizer (){
```

Comment:
This doesn't need to be in an IIFE

Reply:
without it being an IIFE, it never gets called and the value of num is never set to a random number...

Reply:
okay, change made.